bsip-0076.md

```diff
@@ -0,0 +1,88 @@
+    BSIP: 0076
+    Title: Threshold for price feeds through voting
```

Comment:
The polling mechanism was outright disregarded, this was implemented forcefully through coercion/intimidation of price feed publishers in advance of any related worker proposal activating. Inaccurate title, further no subsequent votes have taken place to set accurate feeds in over a month now. Seems highly misleading as a title.